pkg/cli/flags.go

```diff
@@ -36,7 +36,8 @@ const (
 	DockerHubFlagName             = "--docker-hub"
 	DryRunFlagName                = "--dry-run"
 	EnvFlagName                   = "--env"
-	EnvFromFlagName               = "--env-from"
+	EnvValueFromFlagName          = "--env-value-from"
```

Comment:
The corresponding k8s API paths are:

`--env-value-from` -> env:valueFrom:secretKeyRef:...

and

`--env-from-source` -> envFrom:secretRef:name:... where secretRef is a `corev1.EnvFromSource` object

so I think what I have matches the API better, although it's not obvious unless you are  familiar with the API.

Reply:
I'd tend to give the resource paths more credence than the type names. The types are more a part of the implementation than the API. We should also acknowledge that the resource paths are different because they are backed by types, we don't have the same restrictions with cli flags.

It's more important that each flag be clear in the context of the command than that it map directly to the backing data structure.

Reply:
Are you suggesting we use:

```
--env
--envFromValue secretKeyRef:my-secret:key
--envFromSource secretRef:my-secret
```

Reply:
I have 2 alternatives:

1. We could just have a single `--envFrom` but accept either `secretKeyRef:s:k` or `secretRef:s`

2. We could focus on the difference between them being a single value vs all of the secret's key/value pairs being contributed to the env, by emphasizing" value":
```
--envFrom secretRef:my-secret
--envValueFrom secretKeyRef:my-secret:key
```